src/typeDefinitions/default.types.d.ts

```diff
@@ -6,3 +6,10 @@ export interface discordCommand {
   name: string;
   description: string;
 }
+
+export interface responseJson {
```

Comment:
Can we have a better name for this type? 
Can we also have the first letter of type capital, so that they don't look like normal variables?

Reply:
Please suggest a name